BusBoard/BusBoard/Program.cs

```diff
@@ -1,32 +1,29 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
-using Newtonsoft.Json;
 using RestSharp;
 
 namespace BusBoard
 {
     internal class Program
     {
-        public static void Main(string[] args)
+        public static void Main()
         {
             while (true)
             {
-                Console.Write("Enter a bus stop ID: ");
-                var stopId = Console.ReadLine();
-
-                var tflApiUrl = "https://api.tfl.gov.uk";
-
-                var request = new RestRequest($"StopPoint/{stopId}/Arrivals");
-
-                var incomingBusPredictions =
-                    JsonConvert.DeserializeObject<List<IncomingBusPrediction>>(new RestClient(tflApiUrl).Get(request)
-                        .Content);
+                Console.Write("\nEnter a postal code: ");
+                var postCode = Console.ReadLine();
                 
-                incomingBusPredictions = incomingBusPredictions
-                    .OrderBy(t => t.TimeToArrival).ToList();
+                var postCodeCoordinate = new PostCodeApi().GetCoordinate(postCode);
+
+                var nearbyBusStops = new TflApi().GetNearbyBusStops(postCodeCoordinate);
```

Comment:
What if there are no bus stops nearby? Can we add some more error handling here?